index.js

```diff
@@ -475,10 +465,8 @@ const sendUpdates = async (replacement, customMethod) => {
                         errors.add(worked);
                         break;
                 }
-                console.log('finished', p, q, r);
                 missingConnections--;
-                console.log('missing', missingConnections);
```

Comment:
Can you please keep the 2 logs in this case? They're exceptions to the rest of the app (at least for now)